pages/glossary/context.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Context
+key: context
+---
+
+The context of some piece of information is any other information that helps a user understand it.
+
+Both the [visual context](#visual-context) and the [programmatically determined context](#programmatically-determined-context) are part of the context.
+
+**Note**: The maximal "distance" (both visual distance on the device, and tree distance within the DOM tree) between some information and its context depends on the precise situation, and on what kind of information is looked at.
```

Comment:
The note mentions a maximum "distance" but the actual definition doesn't. As written "context" can be a book I read about the subject 4 years ago.

Reply:
The "distance" is used by the "located near it" part of the definition (near being the distance).
I agree that it is only present in the visual context def, and not in the prog. det. one 😞 
I also agree that it is indeed a bit blurry…

Reply:
I added a note in "context" to rule out the book that you have read 4 years ago.